ff/serving/runner/kubernetes.go

```diff
@@ -14,6 +14,15 @@ import (
 
 var namespace string = "default"
 
+type Schedule struct {
+	spec string
```

Comment:
I meant create the string, which is some random archaic string via fields like Days, Hours, etc.

Otherwise every other CronRunner is gonna have to parse the string

Reply:
Changed to what I believe this is addressing in the latest commit